cloudsql/mysql/database-sql/cloudsql.go

```diff
@@ -156,7 +156,16 @@ func mustConnect() *sql.DB {
 
 	// Use the connector when INSTANCE_CONNECTION_NAME (proj:region:instance) is defined.
 	if os.Getenv("INSTANCE_CONNECTION_NAME") != "" {
-		db, err = connectWithConnector()
+		if os.Getenv("DB_USER") == "" && os.Getenv("DB_IAM_USER") == "" {
+			log.Fatal("Warning: One of DB_USER or DB_IAM_USER must be defined")
```

Comment:
NIT: Should this log string be lower case?

Reply:
Good question. I had to check this. It's just error messages that should start with a lowercase letter.